reference/import_export/product-export.rst

```diff
@@ -40,67 +34,46 @@ We can also count a single step named ``export``.
 Product Export Step
 -------------------
 
+Due to the huge refactor of Imports and Exports steps are now services. They are injected in the export job.
```

Comment:
It's not an upgrade doc, it's a cookbook, so imho, no need to mention the change since 1.5, we just explain what dev have to do :) 

We just had warning notes to say it has been refactored in the begining of the chapter for instance.